recipes/nordic/build.sh

```
@@ -18,6 +18,6 @@ cd ${SRC_DIR}"/bonesis-0.5.7/"
 
 "${PYTHON}" -m pip install --no-deps --no-build-isolation . -vvv
 
-cd ${SRC_DIR}"/NORDic-2.4.4/"
+cd ${SRC_DIR}"/NORDic-2.5.0/"
```

Shellcheck

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)